playbooks/amq_broker.yml

```diff
@@ -35,8 +35,14 @@
     # anything inside <!--start {{ <downstream_placeholder>_delete }} --> and <!--end {{ <downstream_placeholder>_delete }} --> will be removed
     downstream_placeholder_delete:
       - build_status
+      - roles_paths
     # anything inside <!--start {{ <downstream_placeholder>_content }} --> and <!--end {{ <downstream_placeholder>_content }} --> will be replaced with content
     downstream_placeholder_content:
+      - placeholder: roles_paths
+        content: |
+          ### Roles:
+
+          * [`activemq`](https://console.redhat.com/ansible/automation-hub/repo/published/redhat/amq_broker/content/role/amq_broker/): Perform installation and configuration
```

Comment:
the replacement happens after subsitutions, so this should be `* [amq_broker]` not `* [activemq]`